### DISCUSSION TITLE
Error: Ghost: No update since  900 [s] for global cluster  20  with other cluster id  20  at state  "Corrected"  mayPredict =  0  mayPredict (steps) =  1  mayCorrect =  1  mayCorrect (steps) =  1  maySync =  0

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#949] [Q&A] [answered]
What causes this error?

**Comment by davschneller** [comment]
How is your clustering file looking like? (should be in the output directory, titled `{meshname}-clustering.csv`)

**Reply by buzhiqiuyuebai** [reply, marked as answer]
This seems to be a strange problem, I changed the value of MPI*OpenMP and the error disappeared!

**Reply by davschneller** [reply]
Great to hear that it worked! It may be still worthwhile to take a look at the `clustering.csv` file, if you still have it around that is—maybe there's some bug in the code that we can fix with that information.
